settings.gradle

```diff
@@ -1,8 +1,11 @@
 rootProject.name = 'dialogue'
 
+include 'dialogue-blocking-channels'
 include 'dialogue-client-test-lib'
 include 'dialogue-core'
 include 'dialogue-example'
+include 'dialogue-httpurlconnection-client'
+include 'dialogue-hc4-client'
```

Comment:
dialogue-apache-hc4? seems a tad over concise

Reply:
I thought hc stood for HttpClient, though that name is overloaded with java these days ;-)

Feign calls it `httpclient` https://github.com/OpenFeign/feign/tree/master/httpclient
As does dropwizard-metrics: https://metrics.dropwizard.io/3.1.0/manual/httpclient/

+1 for `apache-hc4` to disambiguate from other clients.